app/Cart.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App;
+
+
+class Cart
+{
+    public $items = null;
+    public $totalQty = 0;
+    public $totalPrice = 0;
+
+    public function __construct($oldCart)
```

Comment:
Imaging that the user has add the same product more than one, in this case I prefer to increase the quantity of the product instead of repeat the same product again.